src/main/java/com/teamc/fms/entity/BookingTimeslot.java

```diff
@@ -0,0 +1,18 @@
+package com.teamc.fms.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "booking_timeslot")
+public class BookingTimeslot {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private String timeslotId;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private String bookingId;
+}
```

Comment:
both timeslotId and bookingId are FKs so no need for 
```
@Id
@GeneratedValue(strategy = GenerationType.UUID)
```
So the entity should be     
```
@Data
@Entity
@Table(name = "booking_timeslot")
public class BookingTimeslot {

    @ManyToONe
    private TimeSlot timeslot;

    @ManyToOne
    private Booking booking;
}
```